ironfish-cli/src/commands/chain/export.ts

```diff
@@ -18,7 +18,11 @@ export default class Export extends IronfishCommand {
       char: 'p',
       parse: (input: string): Promise<string> => Promise.resolve(input.trim()),
       required: false,
-      default: '../ironfish-graph-explorer/src/data.json',
+      default: (): Promise<string> => {
+        Assert.isNotUndefined(process.env.PWD)
+
+        return Promise.resolve(`${process.env.PWD}/../ironfish-graph-explorer/src/data.json`)
```

Comment:
This PR still uses incorrect relative paths. How about we just set the default path to the data directory in a folder called exports? I would recommend maybe just removing this default entirely, then below use the data dir from `this.sdk.config.dataDir` where the data dir is stored if no path is specified.

Reply:
It's done by second way: default value of flag set to empty string and we will resolve path using `this.sdk.config.dataDir` in that case.